freeipa/src/main/java/com/sequenceiq/freeipa/converter/cloud/StackToCloudStackConverter.java

```diff
@@ -136,14 +152,26 @@ private CloudInstance buildInstance(InstanceMetaData instanceMetaData, InstanceG
         InstanceAuthentication instanceAuthentication = buildInstanceAuthentication(stackAuthentication);
         Map<String, Object> parameters = buildCloudInstanceParameters(environmentCrn, instanceMetaData);
 
-        return new CloudInstance(id, instanceTemplate, instanceAuthentication, parameters);
+        return new CloudInstance(
+                id,
+                instanceTemplate,
+                instanceAuthentication,
+                instanceMetaData.getSubnetId(),
+                stack.getAvailabilityZone(),
```

Comment:
Note: As discussed, this will have to be replaced with `instanceMetaData.getAvailabilityZone()` once the respective change is made to `InstanceMetaData` (freeipa) in a later PR. Nothing to do now.